pkg/osvscanner/vulnerability_result.go

```diff
@@ -47,8 +51,45 @@ func groupResponseBySource(r *output.Reporter, query osv.BatchedQuery, resp *osv
 
 		pkg.Vulnerabilities = response.Vulns
 
-		pkg.Groups = grouper.Group(grouper.ConvertVulnerabilityToIDAliases(pkg.Vulnerabilities))
+		if groupsByIDBySource[query.Source] == nil {
+			groupsByIDBySource[query.Source] = make(map[string]*models.GroupInfo)
+		}
+
+		groups := grouper.Group(grouper.ConvertVulnerabilityToIDAliases(pkg.Vulnerabilities))
+		// TODO(rex): Is there a better way to do this, this feels fragile
+		for i := range groups {
+			for _, id := range groups[i].IDs {
+				groupsByIDBySource[query.Source][id] = &groups[i]
+			}
+		}
+		pkg.Groups = groups
 		groupedBySource[query.Source] = append(groupedBySource[query.Source], pkg)
+		vulnBySource[query.Source] = append(vulnBySource[query.Source], pkg.Vulnerabilities...)
+	}
+
+	if callAnalysis {
+		for si, v := range vulnBySource {
+			if si.Type == "lockfile" && filepath.Base(si.Path) == "go.mod" {
+				res, err := govulncheckshim.RunGoVulnCheck(filepath.Dir(si.Path), v)
+				if err != nil {
+					// TODO: Better error message to notify user that go is not installed
```

Comment:
Can we at least just include "the Go toolchain is required to be installed" or something similar in the error message for now?